### ISSUE TITLE
cannot install distutils package

### ISSUE THREAD

**Issue by stupidzayac** [avalon60/ctk_theme_builder#22]
```
import distutils.core
ModuleNotFoundError: No module named 'distutils'
```
python version 3.11&3.12

**Comment by avalon60** [comment]
I'm afraid I need more detail of how you got to this point.

**Comment by stupidzayac** [comment]
using CTk Theme Builder Release 2.5.1 & Windows10 
i have tried python3.11&3.12 both of them show the same error

**Comment by PhanthomAvatar** [comment]
PS C:\Users\abele> pip list
Package       Version
------------- -------
customtkinter 5.2.2
darkdetect    0.8.0
distlib       0.3.8
filelock      3.13.1
packaging     23.2
pillow        10.2.0
pip           23.3.2
platformdirs  4.1.0
ttkbootstrap  1.10.1
virtualenv    20.25.0
PS C:\Users\abele>
PS C:\Users\abele> python --version
Python 3.11.7
PS C:\Users\abele>

**Comment by avalon60** [comment]
Did you install Python from Windows Store or did you get it from https://www.python.org/downloads/?

**Comment by PhanthomAvatar** [comment]
I have both, just unistalled the microsoft store and tryed again and still

**Comment by avalon60** [comment]
Have you tried uninstalling all of your Python and installing Python 3.10? That's the version I am using. I think distutils is being deprecated from later versions of Python, and this might be the issue.

**Comment by PhanthomAvatar** [comment]
I did unistall everything and installed that one from python web, now ima try installing setuptools and il update you out

**Comment by avalon60** [comment]
Also, please ensure that you are following something like the correct procedure. This may provide some idea: https://github.com/avalon60/ctk_theme_builder/issues/23#issuecomment-1949367942

**Comment by avalon60** [comment]
I believe this should be fixed with 3.0.1.

**Comment by avalon60** [comment]
Closing - no response.
